test/unit/rbtree_test.js

```diff
@@ -20,10 +20,11 @@
 
 var RBTree = require('../../lib/ring/rbtree').RBTree;
 var RingNode = require('../../lib/ring/rbtree').RingNode;
+var comparator = require('../../lib/ring').comparator;
```

Comment:
I think it is best to define the comparator function in the test instead of reusing the one in the code so we don't have to change the tests when we change the key/value in the RBTree for the HashRing.